src/main/java/org/jabref/logic/citationstyle/CSLAdapter.java

```diff
@@ -84,13 +85,22 @@ private void initialize(String newStyle, CitationStyleOutputFormat newFormat) th
          * Converts the {@link BibEntry} into {@link CSLItemData}.
          */
         private static CSLItemData bibEntryToCSLItemData(BibEntry bibEntry) {
+
+            // create a copy of bibEntry
```

Comment:
This comment states the same as it is done in the code. So please "Remove Superfluous Comments" [Java by Comparison](http://java.by-comparison.com/).

You can also describe, *why* a clone is generated!

Reply:
Is this a good comment?
```
/**
         * Converts the {@link BibEntry} into {@link CSLItemData}.
         *
         * Clone bibEntry to make a save changes,
         * change month field from JabRefFormat <code>#mon#</> to ShortName <code>mon</code>
         * because CSL does not support JabRefFormat.
         */
```